solutions/devsprint-firas-safa-1/GitHubApp/Screens/Settings/SettingsView.swift

```diff
@@ -7,6 +7,62 @@
 
 import UIKit
 
+private let cellIdentifier = "SettingsCell"
```

Comment:
we talked about where and how this variable should be..

Reply:
well, even if, (you might be right), this identifier should have been inside the List class and not glabl. The solution you implemented (custom cell with no actual code) works as well. good job.